lib/tasks/reset.rake

```diff
@@ -0,0 +1,25 @@
+Dir['./db/seeds/*.rb'].each {|file| require file }
```

Comment:
I'm really sorry, I know you explained this file in person to us but I don't remember what it does. Can you give me a quick description please?